plugins/variants/src/VcfFeature/util.ts

```diff
@@ -115,7 +115,7 @@ export function getSOAndDescByExamination(ref: string, alt: string) {
   } else if (alt === '<DEL>') {
     return ['deletion', alt]
   } else if (alt === '<INV>') {
-    return ['deletion', alt]
+    return ['inversion', alt]
```

Comment:
this was also incorrectly saying <INV> type features were deletions, which would show up on feature.get('type'). subtle but needing a fix :)